app/src/main/java/com/example/itacademyhw/ImageItem.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.itacademyhw
+
+import android.graphics.Bitmap
+import android.net.Uri
+
+sealed class ImageItem(val fileName: String) {
+    class Internal(
+        fileName: String,
+        val bitmap: Bitmap
+    ) : ImageItem(fileName)
+
+    class External(
+        fileName: String,
+        val contentUri: Uri
+    ) : ImageItem(fileName)
+}
```

Comment:
дабы сохранить приемущества data классов и при этом использовать как sealed можно сделать так 
```suggestion
sealed class ImageItem {
    abstract val fileName: String
    data class Internal(
        override val fileName: String,
        val bitmap: Bitmap
    ) : ImageItem(fileName)

    data class External(
        override val fileName: String,
        val contentUri: Uri
    ) : ImageItem(fileName)
}
```